packages/@aws-cdk/aws-stepfunctions-tasks/test/sagemaker/processing-image/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# scratch is an empty, zero byte image `/` filesystem. We just need to build an image with
+# at least 1 layer for it to be successfully published as an asset for integration tests
+FROM scratch
+COPY hello /
+CMD echo 'Hello World'
```

Comment:
This was the best I could come up with for a shell container that could be 'used' in an integration test. If you are aware of a cleaner method to achieve this still avoids a 3P dependency do let me know (we just need something with at least 1 layer to ensure that we can successfully build and reference a container image from local assets).

Unlike other sagemaker tasks (such as [pre-built training algorithms](https://docs.aws.amazon.com/sagemaker/latest/dg/algos.html)/images), there aren't any built-in processing images that can be used by default so we need to reference an actual image asset here instead of doing something like [referencing a pre-existing algorithm](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-stepfunctions-tasks/test/sagemaker/integ.create-training-job.ts#L39).

I do think adding a shell asset such as this one is better (due to being light-weight and having no external dependencies) than referencing some other test container such as Amazon lightsail's [hello-world](https://gallery.ecr.aws/amazon-lightsail/hello-world) image.